### ISSUE TITLE
terminal window output cut off after install

### ISSUE THREAD

**Issue by nuttdguy** [emmetio/emmet-eclipse#52]
After installing, terminal within eclipse does not show output the full width of terminal window. The output is cut-off, does not show right half of output. 

Verfiied by removing eclipse 3 times, installing each extension and then checking if an extension was the culprit.

**Comment by nuttdguy** [comment]
![emmet_terminal_issue](https://user-images.githubusercontent.com/6646024/26993739-1f36e73a-4d2a-11e7-93d2-8612157538f0.PNG)

**Comment by zskr** [comment]
How was this issue resolved?

**Comment by nuttdguy** [comment]
The problem corrected itself after an update, but the issue returned after I installing a web-eclipse plugin for Angular. I also tried uninstalling web-eclipse, however the problem remained. I even tried to change the settings directly from the installed program - which worked outside eclipse IDE, but not within the IDE.

Now, because I wasn't sure what or if plugin was causing the problem, I closed this issue and just had been using a workaround.

>> mode con: cols=300 

that expands the terminal display width to 300 characters wide.
